go.mod

```diff
@@ -250,3 +250,5 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.14.3 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
 )
+
+replace sigs.k8s.io/gateway-api => /Users/simone/go/src/sigs.k8s.io/gateway-api
```

Comment:
I will remove the `replace` once the GatewayAPI PR https://github.com/kubernetes-sigs/gateway-api/pull/2013 will be merged and Timeouts implementation will be part of `k8sbeta "sigs.k8s.io/gateway-api/apis/v1beta1"`